src/GovUk.Education.ExploreEducationStatistics.Content.Api/Services/DownloadService.cs

```diff
@@ -0,0 +1,48 @@
+﻿using GovUk.Education.ExploreEducationStatistics.Content.Api.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Content.Api.ViewModels;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Api.Services
+{
+    public class DownloadService : IDownloadService
+    {
+        private readonly ApplicationDbContext _context;
+
+        public DownloadService(
+            ApplicationDbContext context)
+        {
+            _context = context;
+        }
+
+        // TODO: Include the list of downloadable files
```

Comment:
Isn't this missing functionality meant to be the purpose of this PR? :)

Reply:
Yeah doesnt want merging yet, was just adding the endpoint, this needs work from DFE-689 completing first.

Reply:
Is this also DFE-551 @jamescutts ?